### PR TITLE
#134 new scale-control example

### PR DESCRIPTION
**Pull request by OpenSteveMap**

### CODE REVIEW

File: examples/scale-control-directive/index.ts [hunk 1]
```diff
@@ -2,114 +2,93 @@
...
+        (mouseup)="handleEvent('mouseup', event);"
+        (mouseover)="handleEvent('mouseover', event);"
+        (mouseout)="handleEvent('mouseout', event);"
+        (mousemove)="handleEvent('mousemove', event);"
+        (positionChange)="handleEvent('positionChange', event);"
+        (displayChange)="handleEvent('displayChange', event);"
+        
+        [position]="getDuplexPropertyByName('position').value"
+        [(display)]="getDuplexPropertyByName('display').value"
+        [opacity]="getDuplexPropertyByName('opacity').value"
```
Comment (atd-schubert) [inline examples/scale-control-directive/index.ts]: Bind property position and opacity 2-way

**Review by atd-schubert — CHANGES_REQUESTED** [review]

File: examples/scale-control-directive/index.ts (hunk 1 above)
Comment (OpenSteveMap) [reply examples/scale-control-directive/index.ts]: is opacity 2-way?

Comment (atd-schubert) [reply examples/scale-control-directive/index.ts]: You are right in this case opacity is an input!

File: examples/scale-control-directive/index.ts [hunk 2]
```diff
@@ -33,13 +33,13 @@ const template: string = `
...
         
-        [position]="getDuplexPropertyByName('position').value"
         [(display)]="getDuplexPropertyByName('display').value"
-        [opacity]="getDuplexPropertyByName('opacity').value"
         
         [metric]="getInputPropertyByName('metric').value"
         [imperial]="getInputPropertyByName('imperial').value"
         [maxWidth]="getInputPropertyByName('maxWidth').value"
+        [position]="getInputPropertyByName('position').value"
+        [opacity]="getDuplexPropertyByName('opacity').value"
```
Comment (atd-schubert) [inline examples/scale-control-directive/index.ts]: Here it is still trying to get from Duplex! I think this will cause an error...

**Review by atd-schubert — CHANGES_REQUESTED** [review]
getDuplexProperty of an Input property is not valid...

File: examples/scale-control-directive/index.ts (hunk 2 above)
Comment (OpenSteveMap) [reply examples/scale-control-directive/index.ts]: i've forgot to change it to input ...

**Review by atd-schubert — APPROVED** [review]

**Pull request merged**